### PR TITLE
Ubuntu test: Including <image> handling and introduction of special cases

### PR DESCRIPTION
**Pull request by DuesselbergAdrian**
-Image handling: Comparison of SVG files excluding <image> tag
-Special cases: ['TH1/optdate.svg','TH1/optdate2.svg','TH1/optdate3.svg','TH2/image.png','Candle/plot.svg','Candle/stack.svg','TCanvas/time.svg','TGeo/image.png','Misc/taxis.svg','RCanvas/raxis.svg']

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -275,12 +318,25 @@ function processURL(url) {
...
+      //Description: Display all special cases
+      if (all_special.length) console.log('ALL SPECIAL', all_special);
+      //----------------------------------------------------------------------Added for special cases
       console.log('No more data to process');
-      console.log('SUMMARY: match', nmatch, 'diff', ndiff, 'new', nnew);
+      console.log('SUMMARY: match', nmatch, 'diff', ndiff, 'new', nnew, 'special', nspecial); // added for special cases
+
+      // Description: If one file pair differs, the test fails
+      if (ndiff > 0) {
+         structuredLogger('ERROR', 'Not all files match', { diffCount: ndiff });
```
Comment (linev) [inline test.js:337]: Are you using 'ERROR' string to catch problem in the jsroot workflow?

Comment (DuesselbergAdrian) [reply test.js:337]: This is the error message, if there is a diff:

> SUMMARY: match 44 diff 1 new 0
> {"level":"ERROR","message":"Not all files match","diffCount":1,"timestamp":"2024-04-18T07:48:59.952Z"}
> Error: Process completed with exit code 1.

I think it's just for printing.

Comment (linev) [reply test.js:337]: Ok, I see.

**Comment by linev** [comment]
I merging changes here and then you need to adjust PR for jsroot

**Pull request merged**